requestHelper.go

```
@@ -0,0 +1,23 @@
+package mux
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func  HandleJson(r *http.Request, params *RequestParams){
+	var data map[string]interface{}
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&data)
+
+	if err != nil {
+		panic(err)
```

I do not think `panic` is the right thing to do. We need to report an error as a response. Panic will shutdown the server. So, hackers can hack make us down easily.